addons/refuel/functions/fnc_checkFuel.sqf

```diff
@@ -21,7 +21,7 @@ params [["_unit", objNull, [objNull]], ["_target", objNull, [objNull]]];
 private _fuel = [_target] call FUNC(getFuel);
 
 [
-    REFUEL_PROGRESS_DURATION * 2.5,
+    TIME_PROGRESSBAR(REFUEL_PROGRESS_DURATION * 2),
```

Comment:
No, this is not flow rate, it's how long it takes to check current fuel level of a vehicle. Macro is 2s, so this makes it 4s.